tendermint/src/lite/types.rs

```diff
@@ -114,7 +114,7 @@ pub trait TrustThreshold {
     }
 }
 
-/// Requester can be used to request `SignedHeaders` and `ValidatorSet`s for a
+/// Requester can be used to request [`SignedHeader`]s and [`ValidatorSet`]s for a
```

Comment:
What's the story with these brackets?

Reply:
Creates links to the types in the docs. Run `cargo +nightly doc --open --no-deps`  :-)